### PR TITLE
job board app: offers list + detail view

### PR DESCRIPTION
**Pull request by ernestoarbitrio**
Job board application:

- create 'only' via admin job offers

- list offers and detail view in a public page

### CODE REVIEW

File: jobboard/templates/jobboard/joboffer_list.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <ul>
+                        {% for offer in object_list %}
+                            <li>
+
+                                <div class="microblog post-summary">
+                                    <h2 class="h4"><a title="Permalink a questa offerta" href="{% url 'job-detail' offer.slug %}">{{ offer.job_title }}</a></h2>
+                                    <div class="summary">
+                                        {{ offer.job_description|truncate_chars:150 }}
+                                    </div>
+{#                                    <div class="meta">#}
```
Comment (yakky) [inline jobboard/templates/jobboard/joboffer_list.html]: Commented section can be removed?

File: jobboard/templates/jobboard/joboffer_list.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                                </div>
+                            </li>
+                            <hr>
+                        {% endfor %}
+                    </ul>
+                </div>
+            </div>
+
+{#            <div class="grid-33">#}
+{#                <div id="blog-archives-browser">#}
```
Comment (yakky) [inline jobboard/templates/jobboard/joboffer_list.html]: Same

File: jobboard/tests.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""
```
Comment (yakky) [inline jobboard/tests.py]: Remove test file if not used

File: jobboard/urls.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+__author__ = 'ernesto'
+from .views import JobList, JobDetail
+from django.conf.urls import patterns, url
+from django.utils.translation import ugettext_lazy as _
+
+
+urlpatterns = patterns(
+    'jobboard.views',
+    url('^board/list/$', view=JobList.as_view(), name='job_list'),
```
Comment (yakky) [inline jobboard/urls.py]: why not just matching the empty string to append to `jobs` ?

File: jobboard/templates/jobboard/joboffer_list.html (hunk 1 above)
Comment (ernestoarbitrio) [reply jobboard/templates/jobboard/joboffer_list.html]: done

File: jobboard/templates/jobboard/joboffer_list.html (hunk 2 above)
Comment (ernestoarbitrio) [reply jobboard/templates/jobboard/joboffer_list.html]: done

File: jobboard/tests.py (hunk 3 above)
Comment (ernestoarbitrio) [reply jobboard/tests.py]: done

File: jobboard/urls.py (hunk 4 above)
Comment (ernestoarbitrio) [reply jobboard/urls.py]: done

**Pull request merged**